specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-11-01/managedClusters.json

```diff
@@ -2828,6 +2832,93 @@
       ],
       "description": "A Kubernetes add-on profile for a managed cluster."
     },
+    "ManagedClusterPodIdentity": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the pod identity."
+        },
+        "namespace": {
+          "type": "string",
+          "description": "Namespace of the pod identity."
+        },
+        "identity": {
+          "$ref": "#/definitions/UserAssignedIdentity",
+          "description": "Information of the user assigned identity."
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current provisioning state of the pod identity.",
+          "enum": [
+            "Assigned",
+            "Updating",
+            "Deleting",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedClusterPodIdentityProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "provisioningInfo": {
+          "readOnly": true,
+          "properties": {
+            "error": {
+              "$ref": "#/definitions/CloudErrorBody",
+              "description": "Pod identity assignment error (if any)."
+            }
+          }
+        }
+      },
+      "required": [
+        "name",
+        "namespace",
+        "identity"
+      ]
+    },
+    "ManagedClusterPodIdentityException": {
+      "properties": {
+        "name": {
```

Comment:
is this readyonly?

Reply:
the `name` / `namespace` / `podLabels` are expected to set the by the user, so they're not readonly